documentation/struct_metadata.md

```diff
@@ -4,6 +4,27 @@ In order to properly find, understand and use geophysical data, standardised enc
 
 Both discovery metadata and use metadata can be embedded in the files produced for a dataset through utilisation of self-explaining file formats. If properly done by the data producer, publication and preservation of data through services is simplified and can be automated.
 
+
+In order to support the FAIR principles, the following guidelines must apply to metadata:
```

Comment:
What about a hybrid version? In my suggestion I added a link because I think it is good to make it clear why metadata should be available if the data is not and because the sentence is the same as the original on the webpage